### PR TITLE
Add note on milestones for PRs

### PR DESCRIPTION
**Pull request by melissawm**
# Description
Adds a small note on adding milestones to PRs. If this is not useful, feel free to close!

## Type of change

- [x] Fixes or improves existing content
- [ ] Adds new content page(s)
- [ ] Fixes or improves workflow, documentation build or deployment

# References
--

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added [alt text](https://webaim.org/techniques/alttext/) to new images included in this PR

### CODE REVIEW

File: docs/developers/core_dev_guide.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ should be covered by at least one test. Good tests not only execute the code,
 but explore corner cases.  It can be tempting not to review tests, but please
 do so.
 
+7. **Milestones:** Make sure you add the appropriate milestone to PRs before
```
Comment (psobolewskiPhD) [inline docs/developers/core_dev_guide.md]:
```suggestion
7. **Labels and Milestones:** Make sure you add the appropriate label and milestone to PRs before
```

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by brisvag** [comment]
We should add that this check is triggered by the `ready_to_merge` label, so adding that is a pretty important step now.

**Comment by melissawm** [comment]
How about this wording?

**Review by brisvag — APPROVED** [review]
